include/gz/math/Inertial.hh

```diff
@@ -123,6 +163,51 @@ namespace gz
         return R * this->massMatrix.Moi() * R.Transposed();
       }
 
+      /// \brief Spatial mass matrix for body B. It does not include fluid
+      /// added mass.
+      /// The matrix is arranged as follows:
+      ///
+      /// | m          0          0          0           m * Pz    -m * Py |
+      /// | 0          m          0          -m * Pz    0           m * Px |
+      /// | 0          0          m           m * Py    -m * Px    0       |
+      /// | 0          -m * Pz     m * Py    Ixx        Ixy        Ixz     |
+      /// |  m * Pz    0          -m * Px    Ixy        Iyy        Iyz     |
+      /// | -m * Py     m* Px     0          Ixz        Iyz        Izz     |
```

Comment:
as noted in https://github.com/gazebosim/sdf_tutorials/pull/76#discussion_r914144751, the spatial matrix convention used by [dartsim](https://github.com/dartsim/dart/blob/c5ed203bba13c427a4525dfd4b903080c2d34af6/dart/dynamics/Inertia.cpp#L409-L425) and [drake](https://drake.mit.edu/doxygen_cxx/group__multibody__spatial__inertia.html) (and based on [Featherstone's convention](https://homes.cs.washington.edu/~todorov/courses/amath533/FeatherstoneSlides.pdf)) uses `[p q r x y z]`, which differs from the `[x y z p q r]` convention used here. What is the plan for translating these matrices? It wouldn't be too hard using the `SubMatrix` API, but it's worth thinking about

Reply:
I was planning on rearranging the terms as needed when the value is passed to the physics engine. We can add helper functions here if you think that would help.

I'm not opposed to swapping things to follow the Featherstone convention, but putting translation above rotation is more in-line with Gazebo and ROS conventions, as well as most of the maritime literature I've looked at.

Reply:
ok, I just wanted to make sure we've thought about it. I think the `SubMatrix` API should be enough